clang/lib/Sema/SemaOpenMP.cpp

```diff
@@ -15901,6 +15901,13 @@ ExprResult SemaOpenMP::VerifyPositiveIntegerConstantInClause(
         << E->getSourceRange();
     return ExprError();
   }
+
+  if (!Result.isRepresentableByInt64()) {
```

Comment:
So these are always 64 bit integers? This: https://www.openmp.org/spec-html/5.2/openmpsu30.html#x60-620004.4.3, "expression of integer type"

Reply:
In our implementation, yes, because we const evaluate the value to APInt and it store the value in i64